setRXFilter missing return values

> ~/Documents/Arduino/libraries/can_common/src/can_common.cpp: In member function 'int CAN_COMMON::setRXFilter(uint8_t, uint32_t, uint32_t, bool)':
> ~/Documents/Arduino/libraries/can_common/src/can_common.cpp:157:1: warning: no return statement in function returning non-void [-Wreturn-type]
> }
> ^
> ~/Documents/Arduino/libraries/can_common/src/can_common.cpp: In member function 'int CAN_COMMON::setRXFilter(uint32_t, uint32_t, bool)':
> ~/Documents/Arduino/libraries/can_common/src/can_common.cpp:162:1: warning: no return statement in function returning non-void [-Wreturn-type]

--- a/src/can_common.cpp
+++ b/src/can_common.cpp
@@ -322,12 +322,12 @@ void CAN_COMMON::removeCallbackFD(uint8_t mailbox)
 
 int CAN_COMMON::setRXFilter(uint8_t mailbox, uint32_t id, uint32_t mask, bool extended)
 {
-    _setFilterSpecific(mailbox, id, mask, extended);
+	return _setFilterSpecific(mailbox, id, mask, extended);
 }
 
 int CAN_COMMON::setRXFilter(uint32_t id, uint32_t mask, bool extended)
 {
-    _setFilter(id, mask, extended);
+    return _setFilter(id, mask, extended);
 }
 
 int CAN_COMMON::watchFor() 

